src/sgame/sg_bot.cpp

```diff
@@ -562,6 +562,10 @@ void G_BotIntermissionThink( gclient_t *client )
 
 bool G_BotInit()
 {
+	// we read but discard the result to register the cvar, so that people
```

Comment:
Not going to work with lazy navmesh initialization...

Reply:
Ugh. This is getting quite annoying.

I've made a new version using Cvar::Modified. How about that?